libs/community/langchain_community/embeddings/gradient_ai.py

```diff
@@ -70,23 +67,26 @@ def validate_environment(cls, values: Dict) -> Dict:
         )
 
         values["gradient_api_url"] = get_from_dict_or_env(
-            values, "gradient_api_url", "GRADIENT_API_URL"
+            values, "gradient_api_url", "GRADIENT_API_URL", default=None
```

Comment:
for what its worth, as currently set up values will always have "https://api.gradient.ai/api" prepopulated for gradient_api_url if it isn't specified

Reply:
I added that retrospectivly. 
I realized, that `default=None` is useless, it also leads to an error, which is raised when the values are not set and `default is None`

Reply:
Fixed.